secboot/secboot.go

```diff
@@ -35,6 +36,26 @@ var (
 	devDiskByLabelDir = "/dev/disk/by-label"
 )
 
+type SealKeyModelParams struct {
+	// The snap model
+	Model *asserts.Model
+	// The set of EFI binary load paths for the current device configuration
+	EFILoadChains [][]string
+	// The kernel command line
+	KernelCmdlines []string
+}
+
+type SealKeyParams struct {
+	// The snap model
+	ModelParams []*SealKeyModelParams
+	// The path to store the sealed key file
+	KeyFile string
+	// The path to authorization policy update data file (only relevant for TPM)
+	PolicyUpdateDataFile string
```

Comment:
I expect these structures not to be the end of the story in terms of parametrizing key sealing across devices, they are good enough for now, but probably for clarity  this  one should alos be called TPMPolicyUpdateDataFile .

Reply:
Renamed as suggested.